### DISCUSSION TITLE
Geräteübersicht?

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#2602] [General]
Gab es dort nicht mal den Hinweis ob das Gerät importiert wurde?
Man konnte nach Adaptern / Importer sortieren?
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/ec7337f7-3727-466f-938a-d0fe8765e311)
in v3.2.0-beta.5 fehlt das?
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/4553a856-049b-4b43-b850-0d3446d4c903)

**Comment by Zefau** [comment]
Ab v3.2 wurde das in die Tags aufgenommen

**Comment by mcuiobroker** [comment]
Dann müsste da ja was drin stehen, oder? Also es wird anscheinend nicht übernommen.
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/60f711aa-6014-473f-b914-760536f3bc2d)

**Comment by Zefau** [comment]
Hast du den Importer neu ausgeführt? Oder würde es von v3.1 nicht übernommen?

**Comment by Zefau** [comment]
Mach mal ein Issue auf bitte
